src/renderer/components/hotbar/hotbar-selector.tsx

```diff
@@ -45,25 +46,59 @@ interface Dependencies {
   openCommandOverlay: (component: React.ReactElement) => void;
 }
 
-const NonInjectedHotbarSelector = observer(({ hotbar, hotbarManager, openCommandOverlay }: HotbarSelectorProps & Dependencies) => (
-  <div className="HotbarSelector flex align-center">
-    <Icon material="play_arrow" className="previous box" onClick={() => hotbarManager.switchToPrevious()} />
-    <div className="box grow flex align-center">
-      <Badge
-        id="hotbarIndex"
-        small
-        label={hotbarManager.getDisplayIndex(hotbarManager.getActive())}
-        onClick={() => openCommandOverlay(<HotbarSwitchCommand />)}
-        tooltip={{
-          preferredPositions: [TooltipPosition.TOP, TooltipPosition.TOP_LEFT],
-          children: hotbar.name,
-        }}
-        className="SelectorIndex"
+const NonInjectedHotbarSelector = observer(({ hotbar, hotbarManager, openCommandOverlay }: HotbarSelectorProps & Dependencies) => {
```

Comment:
Name `NonInjectedHotbarSelector` imply that it props interface for non-injected component should be:

```suggestion
export interface HotbarSelectorProps extends Partial<Dependencies> {...}

const NonInjectedHotbarSelector = observer(({ hotbar, hotbarManager, openCommandOverlay }: HotbarSelectorProps) => {
```